src/platform/packages/shared/kbn-unified-data-table/src/restorable_state.ts

```diff
@@ -7,14 +7,32 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
+import type { InTableSearchRestorableState } from '@kbn/data-grid-in-table-search/src/types';
 import { createRestorableStateProvider } from '@kbn/restorable-state';
+import type { DocumentDiffMode } from './components/compare_documents/types';
 
 type SelectedDocId = string;
 
 export interface UnifiedDataTableRestorableState {
   selectedDocsMap: Record<SelectedDocId, boolean>;
+  isFilterActive: boolean; // show only selected fields
+  pageIndex: number;
+  scrollTop: number;
+  scrollLeft: number;
+  inTableSearch?: InTableSearchRestorableState;
+
+  // comparison mode
```

Comment:
The only state I noticed was missing from comparison mode is the scroll position... That might be a lot to ask given the difficulties around restoring the main grid scroll position though 😅